ompi/attribute/attribute.c

```diff
@@ -687,13 +742,44 @@ int ompi_attr_set_c(ompi_attribute_type_t type, void *object,
 
 
 /*
+ * Front-end function internally called by the C API functions to set an
+ * int attribute.
```

Comment:
I guess I'm confused by this "int" label (in this comment / function name, and all its corresponding declarations / uses) because the MPI_Win intrinsic attributes are not just of type `int`.